preupg/xmlgen/compose.py

```diff
@@ -339,8 +335,8 @@ def get_xml_tree():
         try:
             target_tree = ElementTree.parse(template_file).getroot()
         except IOError:
-            sys.exit('Problem with reading %s file'
-                     % settings.xccdf_template)
+            raise IOError('Error: Problem with reading %s file'
+                          % settings.xccdf_template)
```

Comment:
IOError does not append \n to the end of the message. The final \n is added in /bin/preupg when any Exception is caught.